src/vsphere_cpi/spec/unit/cloud/vsphere/soap_stub_spec.rb

```diff
@@ -8,21 +8,27 @@
   let(:retryable_adapter) { instance_double(VSphereCloud::SdkHelpers::RetryableStubAdapter) }
 
   describe '#create' do
-    it 'returns the SDK Soap Adapter' do
-      expect(VimSdk::Soap::StubAdapter).to receive(:new)
-        .with(
-          'https://some-host/sdk/vimService',
-          'vim.version.version12',
-          http_client,
-        )
-        .and_return(base_adapter)
-      expect(VSphereCloud::SdkHelpers::RetryableStubAdapter).to receive(:new)
-       .with(
-         base_adapter,
-       )
-       .and_return(retryable_adapter)
-
-      expect(soap_stub.create).to eq(retryable_adapter)
+    before(:all) do
+      @old_vc_version = $vc_version
+    end
+    after(:all) do
+      $vc_version = @old_vc_version
+    end
+    context 'when vc_version is set to 7.0' do
```

Comment:
Can we put blank lines between `before`, `after`, `context`s, and `it`s? It's hard to see all the different tests when there aren't blank lines.